packages/framework/src/Resources/config/services.yml

```diff
@@ -157,6 +157,11 @@ services:
       - '%shopsys.domain_config_filepath%'
       - '%shopsys.domain_urls_config_filepath%'
 
+  Shopsys\FrameworkBundle\Component\Domain\Multidomain\MultiDomainEntityClassProvider:
```

Comment:
again, this notation without tilde seems weird to me

Reply:
the tilde is still missing...Please check your additions to the `services.yml` completely so there is no this kind of notation at all